study-builder/studies/pancan/i18n/en.conf

```diff
@@ -713,6 +713,7 @@
 "hint_req_signature": "Full Name is required",
 "hint_req_yes_no": "Please choose yes or no"
 "hint_dob": "Please enter date of birth in MM DD YYYY format",
+"hint_comments_too_long": "Comments too long"
```

Comment:
@MocanaAtBroad  please suggest the correct text for a OTHER_COMMENTS validation hint (both en/es):
`"hint_comments_too_long": "Comments too long"`

Reply:
Maximum length of text exceeded.

Ha excedido el máximo número de caracteres permitidos.